about.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Internal CSS -->
+    <link rel="stylesheet" href="./styles/about.css" />
+    <link rel="stylesheet" href="./styles/button.css" />
+    <!-- External CSS -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <title>About</title>
+  </head>
+  <body>
+    <header>
+      <nav class="mobile-nav">
+        <i class="fa-solid fa-bars" id="hamburger"></i>
+      </nav>
+      <nav class="desktop-nav-2">
+        <ul class="flex-list">
+          <li><a href=""></a><i class="fa-brands fa-facebook-f"></i></li>
+          <li><a href=""></a><i class="fa-brands fa-twitter"></i></li>
+          <li><a href=""></a>English</li>
+          <li><a href=""></a>My Page</li>
+          <li><a href=""></a>Logout</li>
+        </ul>
+      </nav>
+      <nav class="desktop-nav">
+        <a href="index.html" class="nav-logo" ><img src="./images/school-logo.png" alt=""></a>
+        <ul class="flex-list">
+          <li><a href="about.html">About</a></li>
+          <li><a href="#"></a>Program</li>
+          <li><a href="#"></a>Sponsor</li>
+          <li><a href="#"></a>News</li>
+          <li class="last-child bubbly-button"><a href="#"></a>Killy Int'l</li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Your design is not like the design provided by microverse. your logo should be aligned a little bit right but it is totally on the left and the `a tag` should have the same color, font size, and font weight. Kindly adjust it
### your design
![his design](https://user-images.githubusercontent.com/81727988/169626753-b381e2fd-ebf0-48f4-86c5-5e661af78463.png)
### Behance design
![figam](https://user-images.githubusercontent.com/81727988/169626773-6eae9e11-feb1-4ce6-9cc3-88d77ceffb19.png)